api/go.mod

```diff
@@ -3,15 +3,15 @@ module sigs.k8s.io/kustomize/api
 go 1.13
 
 require (
+	github.com/emicklei/go-restful v2.9.6+incompatible // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible
 	github.com/go-openapi/spec v0.19.4
-	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/golangci/golangci-lint v1.19.1
```

Comment:
I guess that dependency of the kustomize on mdrip/blackfriday/golangci-lint is back. 

Have a look at that [kind](https://github.com/kubernetes-sigs/kind/issues/962#issue-508871467). golangci-lint is one of the reasons why they stopped using kustomize.